typhos/tests/test_notes.py

```diff
@@ -0,0 +1,99 @@
+import os
+import shutil
+from pathlib import Path
+
+import platformdirs
+import pytest
+import yaml
+from pytestqt.qtbot import QtBot
+
+from typhos.notes import NOTES_VAR, TyphosNotesEdit
+
+from .conftest import MODULE_PATH
+
+
+@pytest.fixture(scope='function')
+def user_notes_path(monkeypatch, tmp_path: Path):
+    # copy user device_notes.yaml to a temp file
```

Comment:
This feels pedantic but it actually changed my understanding of the comment
```suggestion
    # copy user_device_notes.yaml to a temp file
```